registration_visualizer:
  ros__parameters:
    base_dir: "src/KISS-Matcher-main/cpp/examples/build/data/"
    # NOTE(dkim): These are provided examples of PCD pairs
    # src_pcd_path: "HeLiPR-KAIST05/Aeva.pcd"
    # tgt_pcd_path: "HeLiPR-KAIST05/Ouster.pcd"
    # src_pcd_path: "KITTI00-to-07/kitti00.pcd"
    # tgt_pcd_path: "KITTI00-to-07/kitti07.pcd"
    # src_pcd_path: "KITTI00-to-KITTI360/kitti00.pcd"
    # tgt_pcd_path: "KITTI00-to-KITTI360/kitti360_09.pcd"
    # src_pcd_path: "VBR-Collosseo/test0.pcd"
    # tgt_pcd_path: "VBR-Collosseo/train0.pcd"
    src_pcd_path: "Vel16/src.pcd"
    tgt_pcd_path: "Vel16/tgt.pcd"
    # src_pcd_path: "Vel64/kitti_000540.pcd"
    # tgt_pcd_path: "Vel64/kitti_001319.pcd"
    # src_pcd_path: "VBR-Collosseo/test0.pcd"
    # tgt_pcd_path: "VBR-Collosseo/train0.pcd"

    resolution: 0.2
    moving_rate: 200.0
    frame_rate: 30.0
    scale_factor: 1.0
